src/ResponseMerger/NuxtResponseMerger.php

```diff
@@ -29,8 +34,9 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
 
     // Finally, merge responses and serve them.
     $page = $frontendResponse->getBody()->__toString();
-    $page = preg_replace('/<title(.*)>*<\/title>/', '<title$1>' . strip_tags($data['title']) . '</title>', $page);
-    $page = preg_replace('/<main role="main"(.*)><\/main>/', '<main role="main"$1>' . $data['content'] . '</main>', $page);
+    $attribute_regex = self::ATTRIBUTE_REGEX;
+    $page = preg_replace("/<title$attribute_regex>.*<\/title>/", '<title$1>' . strip_tags($data['title']) . '</title>', $page);
```

Comment:
thx, way better. small remark here still, could we leave the outer () out of $attribute_regex. Then reading this line becomes easy - sicne it's important to see what $1 is matching.